### ISSUE TITLE
dotnet pack + multiple .nuspecs -> "given path's format not supported"

### ISSUE THREAD

**Issue by janni2** [fsprojects/Paket#3111]
### Description
I somehow could not prevent, that there were 2 files created: lib.nuspec + lib.symbols.nuspec
This leads to a simple error in the Paket.Restore.targets file, which prevents me from creating a nuget-file

### Repro steps
* skip (see bugfix)

### Expected behavior
* skip (see bugfix)

### Actual behavior
* skip (see bugfix)

### Known workarounds
* Paket version 5.149.0 (downloaded all the files)
* command line that seems to leads to the error looks like that
`[root]\.paket\paket.exe" fix-nuspecs files "obj\Release\SomeLib.0.1.100.nuspec;obj\Release\SomeLib.0.1.100.symbols.nuspec" project-file "[root]\SomeLib/SomeLib.fsproj`
* reason for `.paket\Paket.Restore.targets(189,5): error : The given path's format is not supported`. is line 184-186 in Paket.Restore.targets, especially `Paths="@(_NuspecFiles)"`

* $(_NuspecFiles) seems to be a list while @(_NuspecFiles)" seems to be a string, thats the reason why the "ConvertToAbsolutePath"-Command failes, when the List has size bigger than one:
https://msdn.microsoft.com/de-de/library/bb882668.aspx
* Changing Paket.Restore.targets in line 184 
from
`Paths="@(_NuspecFiles)"`
to
`Paths="$(_NuspecFiles)"`
made it work
* Not sure, if I'm right. Just guessing in that area. Result now is: I got two nupgks, one with the binaries and one with the sources
* I don't know where the recent .targets file is, but this version currently contains the issue: https://github.com/fsprojects/Paket/blob/master/.paket/Paket.Restore.targets

**Comment by matthid** [comment]
Related:
 - https://github.com/fsprojects/Paket/issues/3012
 - https://github.com/fsprojects/Paket/issues/2689
 - https://github.com/fsprojects/Paket/issues/2830

And probably others.

**Comment by janni2** [comment]
STOP. The ConvertToAbsolutePath works correct as it is. 
The problem is that NuspecFileAbsolutePath contains a semicolon whereas my "fix" (lol) leads to an empty string.

**Comment by janni2** [comment]
Ok, I got unclear results, at least 2 nupkg files. The main one with nearly zero size, the secondary one with symbols. Then I played around to filter one of both nuspec files and surprisingly taking the second one leads to an acceptable result (= 1 nupkg with .dlls and 1 with ignorable garbage)
```
 <ItemGroup>
      <_NuspecFilteredFiles Include="$(AdjustedNuspecOutputPath)\*.symbols.nuspec"/>
 </ItemGroup>

 <ConvertToAbsolutePath Condition="@(_NuspecFiles) != ''" Paths="@(_NuspecFilteredFiles)">
     <Output TaskParameter="AbsolutePaths" PropertyName="NuspecFileAbsolutePath" />
 </ConvertToAbsolutePath>          
```
Sadly I didn't find the sourcecode of the PackTask which could explain, what is actually going on.

So the issue can be closed due to duplicate, maybe this workaround to "include" or "exclude" something to gain "some" result to work with, is useful.
